### DISCUSSION TITLE
Inconsistent syllabic splitting on affixes?

### DISCUSSION THREAD

**Discussion by evgenyfadeev** [aerickt/lapwing-for-beginners#28] [Q&A]
Here I give several examples that I found only in the exercise for the chapter 15.

I put a slash on a splitting that I think is valid, but there is no option with that in a dictionary also I give a word with a different suffix as an illustration.

spons/or - root of the word is spons e.g. responsibility
aud/ition - root is aud - audio
predat/or/y - predation
optim/ist/ic - optimal
occup/ant - preoccupation, occupy
infini/tesim/al - infinite (tesim is possibly a latin suffix - not sure, root is fin or finis)
a/rog/ant - prerogative - split on r is according to ch15 is valid on prefixes. the root is rog from latin rogare
philosoph/er - philosophy

Are these omissions in the dictionary?

A related question - if I were to find issues in dictionaries - to which files should I make pull requests?

**Comment by aerickt** [comment]
Affixes should be used only to modify a root word that can exist by itself in English. I wouldn't consider the examples you've listed above as standalone words (i.e. "spons" and "occup" by themselves cannot be used in sentences).

However, where you draw the line of what is a word and how it may be related to another is completely up to you. Using affixes should be natural and come without hesitation. I don't want to make learning the correct etymology of every word a requirement to learning Lapwing. It's perfectly acceptable to bend the rules in a way that makes sense to you as an individual.

For example, "kit" is not related to the word "kitty" (as far as I can tell from https://en.wiktionary.org/wiki/kit#English), but I might write "kitty" as `KEUT/KWREU` in addition to `KEU/TEU` (the latter outline would be considered the "most correct" in Lapwing).

Therefore, it is acceptable to use affixes in the manner that you've described in your post if it makes sense to you. However, the default Lapwing dictionary prefers affixes to be used on **standalone** words. This unfortunately means that you will have to learn the default Lapwing outlines, or add your own outlines whenever you come across these words; choosing the latter will require you to make a lot of additions.

You can submit pull requests the main branch of https://github.com/aerickt/steno-dictionaries. For these styles of outlines, however, I won't be supporting in the default dictionary. I believe most people would also agree that the root words you've pointed aren't standalone words in English.
